src/Widgets/DataCountWidget/DataCountWidgetClass.ts

```diff
@@ -0,0 +1,12 @@
+import { provideWidgetClass } from 'scrivito'
+
+export const DataCountWidget = provideWidgetClass('DataCountWidget', {
```

Comment:
The commit message is lying 😜

Reply:
[Renamed DataSearchCountWidget to DataSearchWidget](https://github.com/Scrivito/scrivito-portal-app/pull/752/commits/88c0d041af19124b1e1f257edb2f72fcf2ed5e61)

Reply:
😸